podo/PodoCount_code/get_glom_props.py

```diff
@@ -5,6 +5,7 @@
 """
 import numpy as np
 import skimage as sk
+from skimage.morphology import binary_dilation, disk

```

Comment:
Do we need to import these again? as you're using them from `sk`. Please remove if not needed.